docs/faq.md

```diff
@@ -143,9 +143,18 @@ RHCOS/OCP version | RHEL version
 
 ## Q: How do I determine what version of an RPM is included in an RHCOS release?
 
-In the future, the package list for a given OCP release will be displayed in [the OpenShift release controller](https://amd64.ocp.releases.ci.openshift.org/). Currently, only diffs are displayed.
+One normally does not care about a specific RHCOS release, but rather a specific
+_OpenShift release_. In that case, head over to [the release controller's
+page](https://amd64.ocp.releases.ci.openshift.org/) for that release.
```

Comment:
Not a new problem, but maybe we should link to the page for each architecture here.